tests/js-tests/project.json

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
I think you can invoke `git status` to check which files have conflict and resolve the conflicts. I always use [sourcetree](https://www.sourcetreeapp.com/) on Mac. If you use windows, i think you have more tools to resolve it.

And there are some other files have the problem.